librz/util/buf.c

```diff
@@ -337,8 +389,15 @@ RZ_API RzBuffer *rz_buf_new_slurp(const char *file) {
 	return new_buffer(RZ_BUFFER_BYTES, &u);
 }
 
-/// create a new sparse RzBuffer where unpopulated bytes are filled with Oxff
-RZ_API RzBuffer *rz_buf_new_sparse(ut8 Oxff) {
+/**
+ * \brief Creates a sparse buffer.
+ * \param Oxff The byte used to fill unpopulated bytes.
+ * \return Return the new allocated buffer.
+ *
+ * The function creates a new allocated buffer using the RZ_BUFFER_SPARSE back end.
```

Comment:
in general, i would not put the RZ_BUFFER_* in the documentation, because i think that should be a somehow internal detail. We should try to describe what it means to have that type of buffer rather than just using the enum val.